__tests__/schema.ts

```diff
@@ -0,0 +1,33 @@
+import { matchersWithOptions } from 'jest-json-schema';
+
+import accountSchemas from './schemas/account.json';
+import libSchemas from './schemas/lib.json';
+import providerSchemas from './schemas/provider.json';
+import sequencerSchemas from './schemas/sequencer.json';
+
+const ajvKeywords = require('ajv-keywords');
```

Comment:
This can be made into an `import` and the typing issue that arises avoided with an `as any`.